### DISCUSSION TITLE
error: RPC failed; curl 55 Failed sending data to the peers

### DISCUSSION THREAD

**Discussion by odenwar** [community/community#57429] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Select Feature Area

- [X] Issues
- [ ] Projects

### Body

hi i am a hardware developer.
This time, I'm trying to manage the version through GitHub.

Versions are from 0.0.1.T to 0.0.5.T.

First, we want to upload 0.0.1.T to the repository.

A screenshot is shown below.
![image](https://github.com/community/community/assets/80653986/0f589dd4-3a43-47d4-9cd9-e1a7c4c2ac61)

I would like to know why it is not uploading.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @odenwar ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by odenwar** [reply]
Was the post moved?
Will you give me an answer if I wait?

**Reply by queenofcorgis** [reply]
The post was moved. I can't guarantee who or when your question will be answered, but I appreciate your patience. Good luck!
